tests/core-manager.js

```diff
@@ -587,6 +587,16 @@ test('deleteData() deletes relevant core storage files', async (t) => {
         peer2DataStorageDeletedForPeer1,
         'peer 1 no longer has `data` storage for peer 2'
       )
+
+      t.is(
+        db1
+          .select()
+          .from(coresTable)
+          .where(eq(coresTable.namespace, 'data'))
+          .all().length,
+        0,
+        'peer 1 `cores` table has no info about `data` core from peer 2'
+      )
```

Comment:
I think it's worth having a test before the delete to check the opposite of this is true, just to double-check our test logic.